dynawo/sources/Models/Modelica/Dynawo/Examples/DynaFlow/IllustrativeExamples/PhaseShifterTransformer.mo

```diff
@@ -0,0 +1,114 @@
+within Dynawo.Examples.DynaFlow.IllustrativeExamples;
+
+/*
+* Copyright (c) 2024, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
+*/
+
+model PhaseShifterTransformer "Elementary system with one infinite bus, one load, two lines and a Phase Tap Changer Trasnformer"
+  extends Icons.Example;
+
+  // Base calculation
+  final parameter Modelica.SIunits.Impedance ZBASE69_0 = 69.0^2/Dynawo.Electrical.SystemBase.SnRef;
+
+  // gen_01 init values:
+  parameter Dynawo.Types.VoltageModulePu U0Pu_gen_01 = 1.;
+
+  // load_01 init values:
+  parameter Dynawo.Types.ActivePowerPu P0Pu_load_01 = 1.;
+  parameter Dynawo.Types.ReactivePowerPu Q0Pu_load_01 = 0.;
+  final parameter Dynawo.Types.ComplexApparentPowerPu s0Pu_load_01 = Complex(P0Pu_load_01, Q0Pu_load_01);
+
+  // Generator
+  Dynawo.Electrical.Buses.InfiniteBus gen_01(UPu = U0Pu_gen_01, UPhase = 0) annotation(
```

Comment:
Could you also remove the "_" from the components name ?
gen1
load1
bus1
bus2
bus3
PhaseShifter1 instead of shift_controller_01
...